### DISCUSSION TITLE
Archivos de prueba / Test cases

### DISCUSSION THREAD

**Discussion by ljgonzalez1** [IIC2133-PUC/2024-2#20] [Tarea 0] [answered]
¿No habrán por ahí sus input files + outputs esperados como test cases para probar mi código? 👀 (...o cualquier otro tipo de pruebas)
Llegué a la parte de "en teoría funciona", pero todos sabemos que aunque compile, el código jamás funciona como debe a la primera.

**Comment by sebacarrascop** [comment, marked as answer]
están en en anuncio de canvas.
